docs/template_plugin/tests/functional/op_reference/experimental_detectron_roi_feature_extractor.cpp

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <tuple>
+
+#include "base_reference_test.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using namespace reference_tests;
+
+using Attrs = op::v6::ExperimentalDetectronROIFeatureExtractor::Attributes;
+using ExperimentalROI = op::v6::ExperimentalDetectronROIFeatureExtractor;
+
+struct ExperimentalROIFunctional {
+    std::shared_ptr<Function> create_function(const std::vector<Tensor>& ed_inputs,
+                                              const std::vector<Tensor>& results) {
+        // NGRAPH_CHECK(ed_inputs.size() == 2, "Incorrect test case. Number of inputs is not equal to 2.");
+        // NGRAPH_CHECK(results.size() == 2, "Incorrect test case. Number of outputs is not equal to 2.");
+        //
```

Comment:
Why these checks are commented? I think you should remove them if they are not needed.

Reply:
Fixed.